packages/api/lib/testrun.js

```diff
@@ -0,0 +1,15 @@
+class TestCaseRun {
+  constructor (okMessage, number, name) {
+    this.successful = (okMessage === 'ok');
+    this.number = number;
+    this.name = name;
+    this.encodedName = encodeURIComponent(this.name);
+    this.failureMessage = 'TODO ERROR MESSAGE, (e.g. stackoverflow error line 13)';
+  }
+
+  display () {
+    return this.number + ', ' + this.name + ', ' + this.time + ', ' + (this.successful ? '1' : '0');
```

Comment:
for ease of shipping this through a UI, we'll probably want to eventually make this a data-structure, rather than a string, so something like:

```
{
  successes: 3,
  failures: 5,
  timeMs: 4994
}
```

Reply:
This class represents a singular run of a test case, but what you said still applies